xknx/dpt/dpt_2byte_float.py

```diff
@@ -173,6 +173,15 @@ class DPTPartsPerMillion(DPT2ByteFloat):
     unit = "ppm"
 
 
+class DPTAirFlow(DPT2ByteFloat):
+    """DPT 9.009 DPT_Value_AirFlow."""
+
+    dpt_main_number = 9
+    dpt_sub_number = 9
+    value_type = "m³/h"
```

Comment:
This is used for lookup eg. in HA sensor configuration (`type: ...`). Having special characters (or whitespace) in there seems a bit awkward. 

I'd call it `air_flow` or `airflow` - or whatever the Knx specs say. It also has to be unique.

Reply:
you are right... 
i'll change it to "air_flow"